ginisdk/src/androidTest/java/net/gini/android/authorization/EncryptedCredentialsStoreTest.java

```diff
@@ -0,0 +1,139 @@
+package net.gini.android.authorization;
+
+import static android.content.Context.MODE_PRIVATE;
+
+import android.content.SharedPreferences;
+import android.test.AndroidTestCase;
+
+/**
+ * Created by Alpar Szotyori on 08.10.2018.
+ *
+ * Copyright (c) 2018 Gini GmbH.
+ */
+public class EncryptedCredentialsStoreTest extends AndroidTestCase {
```

Comment:
I went with `test` because most of the test files used singular and the test file generator also uses singular and one would have to always correct the generated file name...

Reply:
That's what I guessed. Maybe in a future PR and if there is time for it, we should consider homogenize them.

Reply:
You did it in this PR. Awesome!